packages/core/src/components/table/table-body-row-expandable/readme.md

```diff
@@ -53,11 +53,12 @@ We recommend fitting your content within the table’s natural size whenever pos
 
 ## Properties
 
-| Property   | Attribute  | Description                                                                                                                               | Type      | Default              |
-| ---------- | ---------- | ----------------------------------------------------------------------------------------------------------------------------------------- | --------- | -------------------- |
-| `colSpan`  | `col-span` | In case that automatic count of columns does not work, user can manually set this one. Take in mind that expandable control is column too | `number`  | `null`               |
-| `expanded` | `expanded` | Sets isExpanded state to true or false externally                                                                                         | `boolean` | `undefined`          |
-| `rowId`    | `row-id`   | ID for the table row. Randomly generated if not specified.                                                                                | `string`  | `generateUniqueId()` |
+| Property   | Attribute  | Description                                                                                                                               | Type                   | Default              |
+| ---------- | ---------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ---------------------- | -------------------- |
+| `colSpan`  | `col-span` | In case that automatic count of columns does not work, user can manually set this one. Take in mind that expandable control is column too | `number`               | `null`               |
+| `expanded` | `expanded` | Sets isExpanded state to true or false externally                                                                                         | `boolean`              | `undefined`          |
+| `overflow` | `overflow` | Controls the overflow behavior of the expandable row content                                                                              | `"hidden" \| "scroll"` | `'scroll'`           |
```

Comment:
Could we re-build the project to generate a new README file that reflects the updated prop alternatives ('hidden' | 'auto')?

Reply:
Thanks for pointing this out Nathalie :)